app/views/content/cookies.html.erb

```diff
@@ -1,140 +1,138 @@
-        <li>the pages you visit on ‘<%= service_name %>’</li>
-        <li>how long you spend on each page </li>
-        <li>how you got to the site </li>
-        <li>what you click on while you’re visiting the site </li>
-    </ul>
-    <p class="govuk-body">
-    We don’t allow Google to use or share our analytics data.
-    </p>
-
-    <p id="preferences" class="govuk-body">
-    <%= form_with model: @cookie_preferences, url: @cookie_settings_path, method: :post do |f| %>
-    <%= f.govuk_collection_radio_buttons :consent,
-      [['yes', 'Yes, opt-in to Google Analytics cookies'], ['no', 'No, do not track my website usage']],
-      :first,
-      :last,
-      legend: { text: 'Do you accept Google Analytics cookies?' } %>
-    <%= f.govuk_submit 'Save preferences' %>
-    <% end %>
+    <p class='govuk-body'>
+      We use cookies to make this site work and collect information about how you use our service.
     </p>
 
-    <h2 class="govuk-heading-l" id="how-we-use-cookies">How we use cookies</h2>
-
     <h3 class="govuk-heading-m" id="essential-cookies">Essential cookies</h3>
 
     <p class="govuk-body">
-    Essential ‘<%= service_name %>’ cookies include:
+      Essential ‘<%= service_name %>’ cookies keep your information secure while you use this service. We do not need to ask permission to use them.
     </p>
 
     <table class="govuk-table">
       <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header">
-            Name
-          </th>
-          <th scope="col" class="govuk-table__header">
-            Purpose
-          </th>
-          <th scope="col" class="govuk-table__header">
-            Expires
-          </th>
-        </tr>
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">
+          Name
+        </th>
+        <th scope="col" class="govuk-table__header">
+          Purpose
+        </th>
+        <th scope="col" class="govuk-table__header">
+          Expires
+        </th>
+      </tr>
       </thead>
       <tbody class="govuk-table__body govuk-!-font-size-14">
-        <tr class="govuk-table__row">
-          <td class="govuk-table__cell">
-            _apply_for_postgraduate_teacher_training_session
-          </td>
-          <td class="govuk-table__cell">
-            Stores encrypted session data
-          </td>
-          <td class="govuk-table__cell">
-            When you close your browser
-          </td>
-        </tr>
-        <tr>
-          <td class="govuk-table__cell">
-            <%= "consented-to-#{@application}-cookies" %>
-          </td>
-          <td class="govuk-table__cell">
-            Saves your cookie consent settings
-          </td>
-          <td class="govuk-table__cell">
-            6 months
-          </td>
-        </tr>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">
+          _apply_for_postgraduate_teacher_training_session
```

Comment:
Given the length of this string, maybe we can force a line break in the middle:

```suggestion
          _apply_for_postgraduate<wbr>_teacher_training_session
```

Reply:
👍